src/components/ManualAdd.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from 'react';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import AppBar from 'material-ui/AppBar';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import { Link,  } from 'react-router-dom';
+import { connect } from 'react-redux';
+import * as actions from '../actions';
+
+export class ManualAdd extends Component {
+  constructor(props){
+    super(props);
+    this.newData =["Missing", "Missing", "Missing", "Missing", "Missing", "Missing", "Missing", "Missing", "Missing"];
```

Comment:
it seems like it would be easier to use if doi info were an object instead of an array - what are the pros/cons of switching over?